{"name": "test_02_login[xiaoma@126.com-12345678-用户名或密码错误]", "status": "passed", "attachments": [{"name": "stdout", "source": "f61b5ca4-51f1-4e95-9672-49df57ed5501-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'xiaoma@126.com'"}, {"name": "password", "value": "12345678"}, {"name": "expected", "value": "'用户名或密码错误'"}], "start": 1668672147818, "stop": 1668672152009, "uuid": "f658147a-a664-4206-94ed-c3f284ef9c9e", "historyId": "7723dcf02c9d29bb9e36f44ad663994a", "testCaseId": "6a7e0c93f03755875413ec9c9508852b", "fullName": "test.test_web.test_login.Test_Login#test_02_login", "labels": [{"name": "parentSuite", "value": "test.test_web"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_Login"}, {"name": "host", "value": "xiaoma"}, {"name": "thread", "value": "31684-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_web.test_login"}]}